src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -536,6 +536,11 @@ public ResolvedMicrobeTolerances GetEnvironmentalTolerances(MicrobeSpecies speci
 
         var result = MicrobeEnvironmentalToleranceCalculations.ResolveToleranceValues(tolerances);
 
+        if (!species.IsBacteria)
+        {
+            result.OsmoregulationModifier *= 0.2f;
```

Comment:
Wouldn't this still cause the problem with incorrect tolerances being displayed in the GUI?